R/regression.R

```diff
@@ -1600,31 +1604,31 @@ setChartData <- function(result, output)
     return(result)
 }
 
-removeMissingVariables <- function(data, formula, formula.with.interaction,
-                                   missing.variables, outcome.name, input.formula)
+#' @importFrom stats reformulate update
+#' @importFrom flipU OutcomeName
+removeMissingVariables <- function(data, formula, interaction, missing.variables)
```

Comment:
The 3rd argument used to be a formula and now it is a character of the name of the interaction variable. Also `outcome.name` and `input.formula` are gone. Do we need to update calls to `removeMissingVariables`?

Reply:
It has been updated in `regression.R` at https://github.com/Displayr/flipRegression/blob/SAQ-295/R/regression.R and doesnt appear elsewhere.